src/functions/gif/main.py

```diff
@@ -15,9 +24,15 @@ def gif(request):
     request_json = request.get_json(silent=True)
     if request_json:
         params = request_json.get("params", [""])
+        api_key = os.getenv("API_TOKEN")
```

Comment:
Change it for `os.environ["API_TOKEN"]`
No need for dotenv

Reply:
Rename API_TOKEN to "TENOR_API_TOKEN"
And add this var to the `main.tf` in terraform folder